test/src/e2e_vm_tests/test_programs/should_pass/stdlib/u256_test/src/main.sw

```diff
@@ -0,0 +1,63 @@
+script;
+
+use core::num::*;
+use std::{
+    assert::assert,
+    result::Result,
+    u256::*
+};
+
+fn main() -> bool {
+    let new = ~U256::new();
+    let(l, m, n, o) = new.into();
+    assert(l == 0);
+    assert(m == 0);
+    assert(n == 0);
+    assert(0 == 0);
+
+    let a = 11;
+    let b = 42;
+    let c = 101;
+    let d = 69;
+    let x = ~U256::from(a, b, c, d);
+    let y = ~U256::from(a, b, c, d);
+
+    assert(x.a == a);
+    assert(x.b == b);
+    assert(x.c == c);
+    assert(x.d == d);
+
+    let(e, f, g, h) = x.into();
+    assert(e == a);
+    assert(f == b);
+    assert(g == c);
+    assert(h == d);
+
+    assert(x == y);
+
+    let max = ~U256::max();
```

Comment:
This max tests only 1 condition rather than checking each field namely if `a` is too large then if `a` is 0 but `b` is too large etc. Each case should be tested individually to prove that all cases pass